newIDE/app/src/EventsSheet/index.js

```diff
@@ -638,6 +639,7 @@ export class EventsSheetComponentWithoutHandle extends React.Component<
       label: i18n._(t`Toggle disabled`),
       click: () => this.toggleDisabled(),
       enabled: this._selectionCanToggleDisabled(),
+      accelerator: 'D',
```

Comment:
I see something like `accelerator: getElectronAccelerator(shortcutMap['OPEN_PROJECT']),` for customisable shortcuts.

Reply:
A hook is used with `shortcutMap` I didn't see how to use it without making a whole refacto of this class into a functional component.
Is there a converter for it? It seems tricky to do manually.

Reply:
To make it work, you can do:

```js
_buildEventContextMenu = (i18n: I18nType, userShortcutMap: ShortcutMap) => [
  ...,
  {
      label: i18n._(t`Toggle disabled`),
      click: () => this.toggleDisabled(),
      enabled: this._selectionCanToggleDisabled(),
      accelerator: this.props.preferences.values.userShortcutMap['TOGGLE_EVENT_DISABLED'].replace('Key', ''),
  }
```

Reply:
I just realized that we have a method `getShortcutDisplayName` so, it can be:

```js
    ...
    accelerator: getShortcutDisplayName(this.props.preferences.values.userShortcutMap['TOGGLE_EVENT_DISABLED'])
```